### ISSUE TITLE
Fully featured product edit in admin gui

### ISSUE THREAD

**Issue by andersroos** [makerspace/makeradmin#28]

**Comment by HalfVoxel** [comment]
Isn't it better to just redirect to the webshop's edit functionality? Sure it is nice to be able to do everything from the admin gui, but there's also a big maintenance overhead of having to maintain two separate codebases for editing products. I think currently the one in the webshop is the most full-featured.

**Comment by andersroos** [comment]
This is a pretty old issue, I created it as a reminder that not everything was implemented when removing backbone from the admin code.

To give my view on the more general question I think it would be best if admin and public wasn't separate at all as it was when Chille first started out. In that case we would be able to reuse code and make it easier to have fully fledged functionality in admin and a lighter version as a public variant (using the same code).

But given that we can't agree on what technology to use I think we should keep all admin things in the admin interface and remove it from public. I think it will be super confusing for both users and developers that it is in two places. I also think that it will be a nightmare to maintain with links back and forth. Much of the code will have also have to be implemented twice.
